source/dashboards/actions.markdown

```
@@ -168,6 +173,11 @@ hold_action:
       description: "If supported, listen for voice commands when opening the assist dialog and the `action` is defined as `assist`"
       type: boolean
       default: none
+    entity_id:
+      required: false
+      description: "Overrides the default entity to show when when the `action` is defined as `more-info`"
+      type: string
+      default: none      
```

Markdownlint

Expected: 0 or 2; Actual: 6
Trailing spaces

(MD009, no-trailing-spaces)